content/blog/2024/02/camel-whatsapp-langchain4j/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Integrate your AI models effortlessly with Apache Camel"
+date: 2024-02-19
+draft: true
```

Comment:
I'm not sure on the impact of the `draft` attribute.
When ready to publish I guess it needs to be turned to false